### PR TITLE
remove discovery_modules db table

### LINKED ISSUE #11110: remove `discovery_modules` db table
#11092 removes the `DiscoveryModule` model so we can remove the `discovery_modules`  table from the db any time after the tag for 2019-04-04 (i.e. today)

### PR DESCRIPTION
**Pull request by junngo**
I remove `discovery_modules` db table.
Please let me know if I need to correct any mistakes. 

Fixes #11110

### CODE REVIEW

**Comment by junngo** [comment]
Thanks for reviewing code. I appreciate it.
I'm gonna revise PR again.

**Comment by junngo** [comment]
I fixed it.

File: src/olympia/migrations/1084-remove-discovery-modules-table.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS `discovery_modules`;
```
Comment (eviljeff) [inline src/olympia/migrations/1084-remove-discovery-modules-table.sql]: need a new line at the end of the file (that's the red icons shown on github at the end of the line)

Comment (junngo) [reply src/olympia/migrations/1084-remove-discovery-modules-table.sql]: Thank you @eviljeff . I fixed it again.

**Comment by caitmuenster** [comment]
Thanks for the patch, @junngo! 🎉 Your contribution has been added to our [recognition wiki](https://wiki.mozilla.org/Add-ons/Contribute/Recognition). 

Would you be interested in creating an account on mozillians.org? I'd be happy to vouch for you!

**Pull request merged**